rails/app/controllers/dashboard/stories_controller.rb

```diff
@@ -21,9 +21,12 @@ def new
 
     def create
       authorize Story
-      @story = community_stories.new(story_params)
+      @story = community_stories.new(story_params.except(:media))
 
       if @story.save
+        story_params[:media].each do |media|
+          m = @story.media.create(media: media)
+        end
```

Comment:
Normally, yes. Chatting with @rudokemper we are moving to a multiple step process so media upload happens separately. Partially because we want to add some cool media-related features, partially because we’ve gotten some feedback that having them linked can be frustrating.

I could go either way on implementing the transaction for now and removing it in the future, knowing it’s going to change.